test/extensions/filters/network/common/fuzz/network_readfilter_fuzz_test.cc

```diff
@@ -31,13 +33,18 @@ DEFINE_PROTO_FUZZER(const test::extensions::filters::network::FilterFuzzTestCase
         if (std::find(filter_names.begin(), filter_names.end(), input->config().name()) ==
             std::end(filter_names)) {
           absl::string_view filter_name = filter_names[seed % filter_names.size()];
+          input->mutable_config()->clear_typed_config();
```

Comment:
Is this to allow faster creation of valid configs? Can you add a comment why this is done here (and give some insight why in network_writefilter_fuzz_test it is not).

Reply:
Clearing the config is to allow unpacking of `value` correctly. Imagine that a config was present for an RBAC filter and now some other filter is chosen. Unpacking that config might result into some "funny" values, if it works at all.

I am doing this conditionally now, i.e., only when the filter's name has changed.

Applied the changes to both network_read-/writerfilter_fuzz_tests.